### ISSUE TITLE
ERROR TESTING THE MODEL

### ISSUE THREAD

**Issue by mjantoun** [kkanshul/dock#2]
Hello,

I am trying to test your model. when running "th test.lua <pre-trained model>  <unique_id>" (i input the model name and an id) I am getting the below error:

cannot open </mnt/raid01/krishnas/coco/categories.t7> in mode r  at /tmp/luarocks_torch-scm-1-6347/torch7/lib/TH/THDiskFile.c:673
stack traceback:
	[C]: at 0x7f39611cd460
	[C]: in function 'DiskFile'
	/home/ecelab/torch/install/share/lua/5.1/torch/File.lua:405: in function 'load'
	coco.lua:2: in main chunk
	[C]: in function 'dofile'
	dataset.lua:1: in main chunk
	[C]: in function 'dofile'
	test.lua:4: in main chunk
	[C]: in function 'dofile'
	...elab/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x00405d50

The directory specified in the coco.lua file (/mnt/raid01/krishnas/coco/categories.t7) does not exist. how can I get "categories.t7" for class labels?

Thanks,
Maya

**Comment by kkanshul** [comment]
Hi Maya,

categories.t7 just contain the name of 80 MS COCO classes in their standard order. You can just create it yourself. Also, use https://github.com/cocodataset/cocoapi to install MS COCO API for Lua.
